tuned/gtk/gui_profile_loader.py

```diff
@@ -60,7 +60,7 @@ def set_raw_profile(self, profile_name, config):
 
         profilePath = self._locate_profile_path(profile_name)
 
-        if profilePath == tuned.consts.LOAD_DIRECTORIES[1]:
+        if profilePath != tuned.consts.SYSTEM_PROFILE_DIR:
```

Comment:
I think this could be configurable, maybe a list with system profile dirs? In tuned-gtk.conf? Or tuned-main.conf? It could be subset of profile_dirs that would be treated as as system profile dirs.

Or maybe a better way: Add tuned-gtk.conf and option like profile_save_dir which specifies where to save user profiles.

But this can be handled in later PRs, so LGTM for this PR.

Reply:
I agree. In fact, I have many more refactoring changes prepared for the GUI. To make this PR easily reviewable, I only included the necessary changes that keep the GUI working (more or less) as before.